modules/special-rules/pages/pulsar-x.adoc

```diff
@@ -0,0 +1,24 @@
+= Pulsar (X)
+
+This weapon is unique to the Eldar race, and it demonstrates their mastery of laser weaponry.
+
+The weapon shoots in rapid but powerful bursts, guided by advanced targeting technologies.
+As a result, the weapon can destroy multiple targets in a single salvo.
+
+---
+
+When you make a {pulsar-x} shot:
+
+IMPORTANT: A {pulsar-x} shot results in a salvo of {macro-weapon} shots -- but it still only counts as a single super-heavy weapon.
+The resultant salvo of {macro-weapon} shots does *not* count as multiple super-heavy weapons for placement of Blast markers, suppression, contribution to firefights, or any other game purpose.
```

Comment:
Shouldn't this be a single sentence? Or at least remove the duplication of 'results in a salvo of {macro-weapon} shots'/'The resultant salvo of {macro-weapon} shots'?

Something like this:
```suggestion
IMPORTANT: A {pulsar-x} shot results in a salvo of {macro-weapon} shots -- but it still only counts as a single super-heavy weapon for placement of Blast markers, suppression, contribution to firefights, or any other game purpose.
```

Or:
```suggestion
IMPORTANT: A {pulsar-x} shot results in a salvo of {macro-weapon} shots -- but it still only counts as a single super-heavy weapon.
Therefore a {pulsar-x}  does *not* count as multiple super-heavy weapons for placement of Blast markers, suppression, contribution to firefights, or any other game purpose.
```

Reply:
> Shouldn't this be a single sentence? Or at least remove the duplication of 'results in a salvo of {macro-weapon} shots'/'The resultant salvo of {macro-weapon} shots'?

Yes, I kinda ran out of steam after making several iterations in which I created, moved and recombined two separate notes, and needed to come back at another time with fresh eyes. 😁 It would indeed benefit from further rationalisation.

I think your first suggestion might work well, with some further small tweaks to both our contributions to it:

```suggestion
IMPORTANT: A {pulsar-x} shot results in a salvo of {macro-weapon} shots -- but we treat these collectively as only a single super-heavy weapon for all game purposes (such as placement of Blast markers, suppression and contribution to firefights).
```
It's a bit of a long sentence, but acceptable I think.
---

On a related thought: The 'salvo' effect of {pulsar-x} seems to duplicate the principles we outlined in the 'weapon systems' concept — IIRC it's just that the specific number of {macro-weapon} multiple shots in a {pulsar-x} salvo is variable rather than fixed as would be more usual for other weapon systems.

If so, then I wonder if it might make sense to integrate terminology between the two at some point, and/or make {pulsar-x} explicitly a 'weapon system' as opposed to a 'super-heavy weapon' per se?

(We don't need to do it now, it's just a thought to consider for the future.)

Reply:
> I think your first suggestion might work well, with some further small tweaks to both our contributions to it:

The 'only' in the middle looks slightly odd. Minor suggestion:

```suggestion
IMPORTANT: A {pulsar-x} shot results in a salvo of {macro-weapon} shots -- but we treat these collectively as a single super-heavy weapon for all game purposes (such as placement of Blast markers, suppression and contribution to firefights).
```

----

> On a related thought: The 'salvo' effect of {pulsar-x} seems to duplicate the principles we outlined in the 'weapon systems' concept — IIRC it's just that the specific number of {macro-weapon} multiple shots in a {pulsar-x} salvo is variable rather than fixed as would be more usual for other weapon systems.

Ah crap. :-( 

I've just looked at our current definition for 'weapon systems' and it's out of date – multiple shots from a weapon system are **not** treated as a single super-heavy weapon!

Should I start a new branch for that, to prevent scope creep in this branch?